gpterminator/GPTerminator.py

```diff
@@ -61,18 +62,22 @@ def getConfigPath(self):
         self.config_path = configpath
 
     def loadConfig(self):
-        self.getConfigPath()
-        config = configparser.ConfigParser()
-        config.read(self.config_path)
-        self.config_selected = config["SELECTED_CONFIG"]["ConfigName"]
-        self.model = config[self.config_selected]["ModelName"]
-        self.sys_prmpt = config[self.config_selected]["SystemMessage"]
-        self.cmd_init = config[self.config_selected]["CommandInitiator"]
-        self.save_path = config[self.config_selected]["SavePath"]
-        self.temperature = config[self.config_selected]["Temperature"]
-        self.presence_penalty = config[self.config_selected]["PresencePenalty"]
-        self.frequency_penalty = config[self.config_selected]["FrequencyPenalty"]
-        self.code_theme = config[self.config_selected]["CodeTheme"]
+        try:
```

Comment:
I encountered this kind of bug after pulling the newest code but did not change my config file(e.g. lack of `code theme`)